Podcast/SubscriptionCollectionViewCell.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  SubscriptionCollectionViewCell.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 3/6/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class SubscriptionCollectionViewCell: RecommendedSeriesCollectionViewCell {
```

Comment:
instead of having to subclass, you can just initialize it with an enum: .subscription or .recommended and based on that set the label text